webapp/package.json

```diff
@@ -5,11 +5,13 @@
 	"description": "",
 	"scripts": {
 		"build": "webpack --mode=production",
+		"build:watch": "webpack --mode=production --watch",
 		"pack": "cross-env NODE_ENV=production webpack --config webpack.prod.js",
 		"packdev": "cross-env NODE_ENV=dev webpack --config webpack.dev.js",
 		"watchdev": "cross-env NODE_ENV=dev webpack --watch --progress --config webpack.dev.js",
 		"deveditor": "cross-env NODE_ENV=dev webpack server --config webpack.editor.js",
 		"test": "jest",
+		"lint": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx --ext tsx --ext ts . --quiet --cache",
```

Comment:
Should this be combined with `check` from below? Calling this `lint` probably makes more sense in general, but since we tend to call this `check` in other modules, it might be good to stay consistent with that

Reply:
Yup, it's a good catch, will combine this with check.